tasks/utilities/task_czgenepi_wrangling.wdl

```diff
@@ -18,26 +18,57 @@ task czgenepi_wrangling {
     String county_column_name
 
     # optional inputs
-    String gisaid_id_column_name
     String genbank_accession_column_name
     String sequencing_date_column_name
-    String sample_is_private_column_name
+
+    # setting columns
+    String organism
+    Boolean is_private
 
     # runtime
     Int disk_size = 100
   }
   command <<<
+    export is_private=~{is_private}
+   
     # parse terra table for data
     python3 <<CODE
     import pandas as pd
     import numpy as np
     import re
     import os
 
+    # set a function to grab only the year from the date
+    def year_getter(date):
+      r = re.compile('^\d{4}-\d{2}-\d{2}')      
```

Comment:
Not sure how much date filtering we care to include, but this seems to allow **invalid dates** that are in the _valid date format_, e.g. 2023-13-52